otherlibs/systhreads/st_win32.h

```diff
@@ -422,3 +422,17 @@ value caml_wait_signal(value sigs) /* ML */
   caml_invalid_argument("Thread.wait_signal not implemented");
   return Val_int(0);            /* not reached */
 }
+
+void st_determine_thread_stack_size(void** top_of_stack, size_t* stack_size)
+{
+#ifdef _WIN64
```

Comment:
This totally cryptic code MUST come with a comment and a URL to a page that explains NT_TIB and all that stuff.

But: what's wrong with taking the address of a local variable, or using `__builtin_frame_address(0)` like in the Unix code?